docker/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM centos:7
+MAINTAINER sqre-admin
+LABEL version="0.2.1" description="LSST DM/SQuaRE GitHub snapshot tools" \
+      name="lsstsqre/github-snapshot"
+
+USER root
+RUN yum update -y && \
```

Comment:
Is it a big build time win to `&&` some many statement together?  There is a lot going on here.